script/lib/verify-machine-requirements.js

```diff
@@ -154,9 +154,6 @@ function verifyPython() {
     verifyBinary(
       path.join(process.env.SystemDrive || 'C:', 'Python27', 'python.exe')
     );
-    verifyBinary(
-      path.join(process.env.SystemDrive || 'C:', 'Python37', 'python.exe')
-    );
```

Comment:
Why remove this?

Reply:
it's duplicate code, see lines before that

Reply:
Nope `'Python27'` !== `'Python37'`

Reply:
Oh... I didn't see that